assets/src/edit-story/components/canvas/canvasUploadDropTarget.js

```diff
@@ -49,17 +50,30 @@ const PageAreaCover = styled(PageArea)`
 
 function CanvasUploadDropTarget({ children }) {
   const { uploadFile } = useUploader();
+  const { showSnackbar } = useSnackbar();
   const insertElement = useInsertElement();
   const onDropHandler = useCallback(
     (files) => {
       files.forEach((file) => {
-        uploadFile(file).then((res) => {
-          const resource = getResourceFromUploadAPI(res);
-          insertElement(resource.type, { resource });
-        });
+        uploadFile(file)
+          .then((res) => {
+            const resource = getResourceFromUploadAPI(res);
+            insertElement(resource.type, { resource });
+          })
+          .catch((e) => {
+            if (!e.isUserError) {
+              e.message = __(
+                'Sorry, files has failed to upload',
```

Comment:
```suggestion
                'Sorry, files have failed to upload',
```

Reply:
Fixed. It was supposed to be `file has failed...` 😄